### ISSUE TITLE
Embed multiple tracking codes

### ISSUE THREAD

**Issue by thanhansoft** [jehna/ga-lite#14]
HI,
I want to embed ID UA-71257141-1 and UA-71257141-10 to my web

**Comment by jehna** [comment]
That's currently not possible, so I'm flagging this as an enhancement.

**Comment by thanhansoft** [comment]
Hi, I donot know how to embed multi tracking your code. But this is my code. It working fine good my website:

<script type="text/javascript">(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','//www.google-analytics.com/analytics.js','ga');ga('create','UA-67258135-10','auto');ga('send','pageview');ga('create','UA-71257141-1','auto','fdola');ga('fdola.send','pageview');</script>

UA-71257141-1: Tracking code 1
UA-67258135-10: Tracking code 2

I Hope help code you

**Comment by jehna** [comment]
The real GA creates multiple instances with `ga('create', 'UA-XXXXX-Y')`, and I can see that it's not using the `new GA()` syntax because we cannot ensure the library is loaded on startup.

Probably the implementation in our case would be something like:

```
galite.UA = {
  default: 'UA-XXXXXX',
  someName: 'UA-XXXXXX-1'
}
```

For now we would be able to pull off with just a plain array, but the #7 does need the separation with some kind of identifiers.

**Comment by jehna** [comment]
So we should probably somehow separate the [URL building][1] into its own function and use the `eventBuilder` function to loop trough the UA codes for now.

[1]:https://github.com/jehna/ga-lite/blob/09215662a54b50957fa38620e80cee9265966bdd/src/ga-lite.js#L17-L52

**Comment by jehna** [comment]
Probably could mimic the official ga's code and create the instances with `ga()` function, so #7 is also related to this

**Comment by jehna** [comment]
Probably the function would be called `galite()`

**Comment by jehna** [comment]
Aaaand here's the sample code for the official analytics:

```javascript
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-XXXXXXXX-X', 'auto');
ga('send', 'pageview');
```

**Comment by eriktelepovsky** [comment]
Any update to this?

**Comment by jehna** [comment]
@eriktelepovsky I have pretty good picture on how to implement this feature, but I don't have time to work on this project just now. Any help on the implementation would be much appreciated :)

**Comment by jehna** [comment]
Implemented in `version-2.0` branch
